stylesheet.css

```diff
@@ -0,0 +1,101 @@
+@import url("https://fonts.googleapis.com/css2?family=Orbitron&display=swap");
```

Comment:
The only third party thing you are allowed to use is jquery, please remove this front.